templates/package/content/debian.tmpl

```diff
@@ -0,0 +1,65 @@
+{{if eq .PackageDescriptor.Package.Type "debian"}}
+	<h4 class="ui top attached header">{{.locale.Tr "packages.installation"}}</h4>
+	<div class="ui attached segment">
+		<div class="ui form">
+			<div class="field">
+				<label>{{svg "octicon-terminal"}} {{.locale.Tr "packages.debian.registry"}}</label>
+				<div class="markup"><pre class="code-block"><code>sudo curl <gitea-origin-url data-url="{{AppSubUrl}}/api/packages/{{$.PackageDescriptor.Owner.Name}}/debian/repository.key"></gitea-origin-url> -o /etc/apt/trusted.gpg.d/gitea-{{$.PackageDescriptor.Owner.Name}}.asc
+echo "deb <gitea-origin-url data-url="{{AppSubUrl}}/api/packages/{{$.PackageDescriptor.Owner.Name}}/debian"></gitea-origin-url> &lt;distribution&gt; &lt;component&gt;" | sudo tee -a /etc/apt/sources.list.d/gitea.list
+sudo apt update</code></pre></div>
```

Comment:
I would not add `sudo` to the docs here. Personally I prefer to work on a root shell, so it's useless to me. Users should know when to `sudo` (which btw is not even installed in some OS).

Reply:
If you remove the  `sudo`, then at least the text should mention that you need a root shell. When I encounter CLI commands in documentation, I expect them to work when I copy-paste them (except when there are obvious placeholders in there that I need to fill in). 
Maybe preface the command line by "Run the following as root" or something similar? Then it is up to the user to decide how to become root (su, sudo, ...)

Reply:
I guess apt could be configured to run rootless, in which case a `sudo` would not make sense. I think this is a such basic linux thing, we don't need to teach users on this page.

Reply:
Guess the sudo discussion is mostly pointless and will not come to a conclusion, so let's keep it as-is.